eclass/cmake.eclass

```diff
@@ -363,9 +363,9 @@ cmake_src_prepare() {
 		die "FATAL: Unable to find CMakeLists.txt"
 	fi
 
-	# if ninja is enabled but not installed, the build could fail
+	# if NINJA_DEPEND is set and ninja is enabled, but not installed the build could fail
 	# this could happen if ninja is manually enabled (eg. make.conf) but not installed
-	if [[ ${CMAKE_MAKEFILE_GENERATOR} == ninja ]] && ! has_version -b dev-util/ninja; then
+	if [[ ${CMAKE_MAKEFILE_GENERATOR} == ninja ]] && [[ -n "${NINJA_DEPEND}" ]] && ! has_version -b "${NINJA_DEPEND}"; then
```

Comment:
I would just drop this check. BDEPEND should suffice to ensure that ninja or samurai is installed.

Reply:
Sure, I removed it.